custom/icds_reports/templates/icds_reports/icds_app/mobile/navigation.directive.html

```diff
@@ -115,14 +115,15 @@
                     <a href="{$ goToStep('demographics/lactating_enrolled_women', $location.search()) $}" class="sub-links" ng-click="closeMenu()">
                       Lactating Mothers enrolled for Anganwadi Services</a>
                 </div>
+              <!-- This div item will be removed after QA -->
                 <div class="navigation-font pl-3 pr-3 mb-0"
                      ng-if="haveAccessToFeatures==false"
                      ng-class="{'active-mobile-link': $location.path().indexOf('/demographics/adolescent_girls') !== -1}">
                     <a href="{$ goToStep('demographics/adolescent_girls', $location.search()) $}" class="sub-links" ng-click="closeMenu()">
                       Adolescent Girls enrolled for Anganwadi Services</a>
                 </div>
               <div class="navigation-font pl-3 pr-3 mb-0"
-                     ng-if="toShowOnNavigation()"
+                     ng-if="haveAccessToFeatures==true"
```

Comment:
nit: is `==true` necessary here?

Reply:
not really technically, but added just to make the code clearer.

Reply:
interesting. i think not including it is clearer, though suppose is subjective